### PR TITLE
Added doctest to detecting_english_programmatically.py

### PR DESCRIPTION
**Pull request by Suyashd999**
### Describe your change:

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Add or change doctests? -- Note: Please avoid changing both code and tests in a single pull request.
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

**Comment by zhudingsuifeng** [comment]
您好，邮件已经收到，谢谢

**Comment by wwowowo** [comment]
您好，您的邮件已经收到。我会尽快处理，谢谢！

**Comment by Suyashd999** [comment]
why are people replying in Chinese to this PR? even if they are, the translation doesn't makes sense at all!

File: strings/detecting_english_programmatically.py [hunk 1]
```diff
@@ -25,6 +25,18 @@ def get_english_count(message: str) -> float:
 
 
 def remove_non_letters(message: str) -> str:
```
Comment (imSanko) [inline strings/detecting_english_programmatically.py:27]: Nice✅

**Review by imSanko — APPROVED** [review]

**Comment by Suyashd999** [comment]
@cclauss can you review this PR? I have reached the limit of 5 open PRs

**Pull request merged**